src/Form/input-classifiers.js

```diff
@@ -188,9 +188,17 @@ const fourDigitYearRegex = /(\D)\1{3}|\d{4}/i
  * @param {RegExp} regex
  * @returns {RegExpMatchArray|null}
  */
-const findInPlaceholderAndLabels = (input, regex) =>
-    input.placeholder.match(regex) ||
-    [...input.labels].find((label) => label.innerText.match(regex))
```

Comment:
We had a bug in here, the `find` was finding the labels, while the calling function was expecting a `RegExpMatchArray` object.

Reply:
Interestingly, no. Maybe there was some kind of type inference going on behind the scenes.